src/js/components/Table/doc.js

```diff
@@ -0,0 +1,40 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export default (Table) => {
+  const DocumentedTable = describe(Table)
+    .availableAt(getAvailableAtBadge('Table'))
+    .description('A table of data organized in cells.'
+    ).usage(
+      `import { Table, TableCell } from 'grommet';
+<Table />`
+    );
+
+  DocumentedTable.propTypes = {
+    caption: PropTypes.string.description('One line summary'),
+    columns: PropTypes.arrayOf(PropTypes.shape({
+      basis: PropTypes.oneOf(
+        ['xxsmall', 'xsmall', 'small', 'medium', 'large', 'xlarge',
+          '1/2', '1/3', '2/3', '1/4', '3/4']
+      ),
+      dataHeader: PropTypes.bool,
+      footer: PropTypes.node,
+      header: PropTypes.node,
+      label: PropTypes.string,
+      property: PropTypes.string,
+      renderData: PropTypes.func,
```

Comment:
can we move renderData to be a children of Table ? like the children as a function? I know we are using that in other grommet v2 stuff.

Reply:
How would that work, since the data is multi dimensional?